scripts/private-github-packages.test.ts

```
@@ -0,0 +1,498 @@
+  options: SpawnOptions
+) => CommandResult;
+
+type WorkflowStep = {
+  readonly env?: Record<string, string>;
+  readonly name?: string;
+};
+
+type WorkflowPermissions =
+  | Readonly<Record<string, string>>
+  | 'read-all'
+  | 'write-all';
```

ESLint

[error] Replace `|·Readonly<Record<string,·string>>⏎··|·'read-all'⏎·` with `Readonly<Record<string,·string>>·|·'read-all'`

(prettier/prettier)